extension/src/popup/views/AddAccount/ImportAccount/index.tsx

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Form, Field, FieldProps, Formik } from "formik";
+import { bool as YupBool, object as YupObject, string as YupString } from "yup";
+
+import { ROUTES } from "popup/constants/routes";
+import { METRIC_NAMES } from "popup/constants/metricsNames";
+import { AppDispatch } from "popup/App";
+import { navigateTo } from "popup/helpers/navigate";
+import { emitMetric } from "helpers/metrics";
+import { PopupWrapper } from "popup/basics/PopupWrapper";
+import { BottomNav } from "popup/components/BottomNav";
+import { importAccount, authErrorSelector } from "popup/ducks/accountServices";
+
+import { Button, Checkbox, Input, InfoBlock } from "@stellar/design-system";
+
+import "./styles.scss";
+
+export const ImportAccount = () => {
+  interface FormValues {
+    password: string;
+    privateKey: string;
+    authorization: boolean;
+  }
+
+  const initialValues: FormValues = {
+    password: "",
+    privateKey: "",
+    authorization: false,
+  };
+
+  const ImportAccountSchema = YupObject().shape({
+    privateKey: YupString().required(),
+    password: YupString().required(),
+    authorization: YupBool().oneOf([true], "required"),
+  });
+
+  const dispatch: AppDispatch = useDispatch();
+  const authError = useSelector(authErrorSelector);
+
+  const handleSubmit = async (values: FormValues) => {
+    const { password, privateKey } = values;
+
+    const res = await dispatch(importAccount({ password, privateKey }));
+
+    if (importAccount.fulfilled.match(res)) {
+      emitMetric(METRIC_NAMES.accountScreenImportAccount, {
+        number_of_accounts: res.payload.allAccounts.length,
+      });
+      navigateTo(ROUTES.account);
+    }
+  };
+
+  return (
+    <>
+      <PopupWrapper>
+        <div className="ImportAccount">
+          <InfoBlock variant={InfoBlock.variant.warning}>
+            <div className="ImportAccount__warning-header">
+              read before importing your key
+            </div>
+            <div className="ImportAccount__warning-copy">
+              <p>
+                Freighter can't recover your imported secret key using your
+                backup phrase. Storing your secret key is your responsability.{" "}
+              </p>
+              <p>
+                Freighter will never ask for your secret key outside of the
+                extension.
+              </p>
+            </div>
+          </InfoBlock>
+          <Formik
+            initialValues={initialValues}
+            onSubmit={handleSubmit}
+            validationSchema={ImportAccountSchema}
+          >
+            {({ dirty, isSubmitting, isValid }) => (
+              <Form>
+                <div className="ImportAccount__form-row">
```

Comment:
I can change these to use the new FormRows components from: https://github.com/stellar/freighter/pull/378 after merged